website/src/content/api-reference/deployment-previews.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
+
+</Subtitle>
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application in the Merchant Center.
+
+The process involves:
+
+1. Building deployment previews for your Custom Application.
+2. Deploying the Custom Application preview to your hosting provider.
+3. Adding the deployment preview to the Custom Application settings.
+
+# Building deployment previews
+
+Each deployment preview has its unique and stable deployment URL. To build a deployment preview, you must set the `env.production.url` value in the `custom-application-config.json` file to the deployed application. You can follow one of the below approaches:
+
+## Using environment variables
+
+To configure your application URL, you can use environment variables. Then, you can assign different values to the specified environment variable in each deployment environment. The `mc-scripts build` script will automatically resolve the application URL from the environment variables.
+
+To implement this process, create a `.env` file in your project root with the following content:
+
+```bash
+URL=<stable-deployment-url> # copied from your hosting provider
+```
+
+Then, update the `.env.production.url` value in `custom-application-config.json` file to point to the `URL` environment variable.
+
+```json
+{
+  // ...more configurations
+  "env": {
+    "production": {
+      "url": "${env:URL}",
+    },
+  // ...more configurations
+}
+```
```

Comment:
This is messing up the syntax highlighting, as comments are not supported in JSON. Please remove them.

````suggestion
```json title="custom-application-config.json"
{
  "env": {
    "production": {
      "url": "${env:APPLICATION_URL}",
    }
  }
}
```
````